proto/src/determined/api/v1/workspace.proto

```diff
@@ -210,6 +214,51 @@ message DeleteWorkspaceResponse {
   bool completed = 1;
 }
 
+// Request for modifying a namespace binding to a workspace.
+message AddWorkspaceNamespaceBindingRequest {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "id" ] }
+  };
+
+  // The unique id of the workspace.
+  int32 id = 1;
```

Comment:
Can we change this to workspace_id? It would be more clear

Reply:
Good idea, changed!